phpunit.xml.dist

```diff
@@ -38,6 +38,10 @@
                 <directory>./src/Symfony/Bundle/*/Resources</directory>
                 <directory>./src/Symfony/Component/*/Resources</directory>
                 <directory>./src/Symfony/Component/*/*/Resources</directory>
+                <directory>./src/Symfony/Bridge/*/vendor</directory>
+                <directory>./src/Symfony/Bundle/*/vendor</directory>
+                <directory>./src/Symfony/Component/*/vendor</directory>
+                <directory>./src/Symfony/Component/*/*/vendor</directory>
```

Comment:
in fact not exactly what you describe: I sometimes test one single component and forget the vendor there, then later on run the main test suite and it fails becuse phpunit goes into this sub-vendor directory, finds tests to run and boom.